lib/lambda/api/index.ts

```diff
@@ -1,8 +1,24 @@
 import { APIGatewayProxyEventV2, APIGatewayProxyResultV2, APIGatewayProxyStructuredResultV2, Context, Handler } from 'aws-lambda';
+import {route} from '../router'
```

Comment:
```suggestion
import { route } from '../router'
```
Keep your style consistent.